src/App.tsx

```diff
@@ -1,39 +1,66 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
 
 function getRandomName(): string {
   const value = Date.now().toString().slice(-4);
 
   return `Clock-${value}`;
 }
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+export class App extends React.Component {
+  state = {
+    hasClock: true,
+    clockName: 'Clock-0',
+  };
 
-  // This code starts a timer
-  const timerId = window.setInterval(() => {
-    clockName = getRandomName();
-  }, 3300);
+  timerId = 0;
 
-  // this code stops the timer
-  window.clearInterval(timerId);
+  componentDidMount(): void {
+    document.addEventListener('click', this.showClock.bind(this));
+    document.addEventListener('contextmenu', this.hideClock.bind(this));
 
-  return (
-    <div className="App">
-      <h1>React clock</h1>
+    this.timerId = window.setInterval(() => {
+      this.setState((prev) => ({
+        ...prev,
+        clockName: getRandomName(),
+      }));
+    }, 3000);
```

Comment:
Set the correct interval to pass the tests
![image](https://github.com/mate-academy/react_clock/assets/96409563/f706007c-61d3-4a94-ab22-d92cc15e6c49)
![image](https://github.com/mate-academy/react_clock/assets/96409563/85dfa70e-1cf8-449e-93b6-3dbcaf45f728)